### DISCUSSION TITLE
someone know the resolution this error in terminal ubuntu 22.04 (docker)

### DISCUSSION THREAD

**Discussion by marehead** [community/community#31018] [Other Feature Feedback, Questions, & Ideas] [answered]
Cannot connect to the Docker daemon at tcp://0.0.0.0:1273. Is the docker daemon running?

**Comment by airtower-luna** [comment, marked as answer]
Clear error: `0.0.0.0` isn't a valid address to *connect* to, you'd need to use the actual server address. It is valid as a *listening* address, where it means "any available IPv4 address".

However, it's at least odd that you'd connect to Docker over TCP on Ubuntu. Usually that works over a Unix socket on Linux.
